### PR TITLE
Suggested to create branch when subtree pull

### PR DESCRIPTION
**Pull request by RosaWagner**
cc @vv-monsalve @chrissimpkins

### CODE REVIEW

File: gf-guide/googlefonts.md [hunk 1]
```diff
@@ -112,9 +112,11 @@ Google Fonts supports all the [Microsoft registered axes](https://docs.microsof
 This directory is a git subtree that is automatically updated with changes in the [googlefonts/axisregistry](https://github.com/googlefonts/axisregistry) repository.  The contents of this directory should not be changed in commits that are pushed to the [google/fonts](https://github.com/google/fonts) repository.  Rather, make edits in a pull request on the [googlefonts/axisregistry](https://github.com/googlefonts/axisregistry) repository.  [google/fonts](https://github.com/google/fonts) repository maintainers can then pull the git subtree changes in to the `axisregistry` directory by executing the following command from the root of [google/fonts](https://github.com/google/fonts):
 
 ```code
-git subtree pull --prefix=axisregistry https://github.com/googlefonts/axisregistry.git [new branch name]
+git subtree pull --prefix=axisregistry https://github.com/googlefonts/axisregistry.git main
 ```
 
+Once the main branch from the submodule is pulled, you will need to push in a new branch upstream.
```
[the remark this answers is not in the document]
Comment (RosaWagner) [reply gf-guide/googlefonts.md:118]: PR to upstream/new-branch, like we do with fonts

[the remark this answers is not in the document]
Comment (vv-monsalve) [reply gf-guide/googlefonts.md:118]: Exactly, as we do in fonts. Makes sense. More of a safe process.

[the remark this answers is not in the document]
Comment (vv-monsalve) [reply gf-guide/googlefonts.md:118]: But then the process would be: pull google/fonts main --> checkout -b new-branch --> git pull subtree --> push it to google/fonts/new-branch --> PR new-branch--to--main, right?

How about:

Ensure updating your local main before running the git pull subtree command. Once the main branch from the submodule is pulled, you will need to push in a new branch upstream and create a PR from it to `main`.

[the remark this answers is not in the document]
Comment (RosaWagner) [reply gf-guide/googlefonts.md:118]: First process described sounds correct.

[the remark this answers is not in the document]
Comment (vv-monsalve) [reply gf-guide/googlefonts.md:118]: From it is the suggestion to the paragraph.

**Review by chrissimpkins — APPROVED** [review]

**Pull request merged**